### ISSUE TITLE
[Tab Extended]: QA: Text elements are receiving keyboard focus

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9233]
### Description

Page Element: Text elements

Issue Description:
Text elements are receiving keyboard focus.

Expected:
Only interactive elements should receive keyboard focus.

User Impact:
When standard methods of keyboard navigation is not supported, this impact the user ability to interact with the page controls. This impacts Keyboard only users/Screen reader users

### Component(s) impacted

Tab extended
Tab extended with media

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component canary: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
